src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: block;
+    box-sizing: border-box;
+    margin: 0;
+    margin-right: 4px;
+    padding: 0;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &--1 .stars__star:nth-child(-n + 4) {
+    background-image: url(/images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+}
```

Comment:
Here you created only 1 modifier for 1 active star. For the rest (2,3,4,5 stars) there is no required behavior. 
You should use `for` and replace values with iterator value:
&--**1** .stars__star:nth-child(-n + **4**)